oauth2_provider/locale/pt/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,172 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
I guess these lines could use some updating. Unless you'd rather delete them completely,

Reply:
I'll just remove.